src/Umbraco.Core/IO/CleanFolderResult.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Umbraco.Core.IO
+{
+    public enum CleanFolderResultStatus
+    {
+        Success,
+        FailedAsDoesNotExist,
+        FailedWithException
+    }
```

Comment:
I think this should be in its own file..

Reply:
Agreed, have moved.